apkbuild-cpan.in

```diff
@@ -14,8 +14,9 @@ use Module::CoreList;
 use JSON;
 
 my $license_mappings = {
-	"perl_5" => "GPL PerlArtistic",
+	"perl_5" => "GPL-1.0-or-later Artistic-1.0-Perl",
 	"artistic_2" => "Artistic-2",
```

Comment:
[Artistic-2.0](https://spdx.org/licenses/Artistic-2.0.html) ?

Reply:
Yes, you are correct.  I have squashed and pushed an update